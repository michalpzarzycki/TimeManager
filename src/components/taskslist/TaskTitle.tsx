import React from 'react';
import styles from './TaskTitle.module.css'

export default function TaskTitle({task}: any) {
    return(
        <div className={styles.title}>{task.task}</div>
    )
}