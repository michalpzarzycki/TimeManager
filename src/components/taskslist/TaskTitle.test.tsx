import React from 'react'
import {shallow} from 'enzyme'
import TaskTitle from './TaskTitle'

test('Expext to render TaskTitle component', () => {
    expect(shallow(<TaskTitle task={{}}/>)).toMatchSnapshot()
})
