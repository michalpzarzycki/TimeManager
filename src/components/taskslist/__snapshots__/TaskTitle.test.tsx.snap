// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expext to render TaskTitle component 1`] = `ShallowWrapper {}`;
